build_windows.md

```diff
@@ -16,17 +16,183 @@ Pandoc could be installed from https://pandoc.org/installing.html When performin
 "Install for all users" (to put the binaries into "Program Files" directory).
 
 ## Install WiX Toolset v3 (is preinstalled in GitHub runner)
-The latest release of the WiX Toolset can be obtained from https://wixtoolset.org/docs/wix3/.  Installing it into a clean VM might require
+The latest release of the WiX Toolset can be obtained from https://wixtoolset.org/docs/wix3/. Installing it into a clean VM might require
 an additional installation of .NET Framework 3.5 in advance
 ([instructions for adding .NET Framework 3.5 via enabling the Windows feature](https://learn.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows#enable-the-net-framework-35-in-control-panel))
 
-## Install msys2
+## Building and running without MSYS2
+
+### Install git and go
+
+To build podman you need [git](https://gitforwindows.org/) and [go](https://go.dev). In case you haven't them yet you can 
+install them opening a Windows Terminal and run the following command:
+
+```
+winget install -e GoLang.Go Git.Git
+```
+
+:information_source: For the `PATH` to be reloaded, a restart of your terminal is advised.
+
+### Enable Hyper-V (optional)
+
+Podman on Windows can run on the [Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/windows/wsl/) or 
+on [Hyper-V](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v) . If you want to 
+use Hyber-V and you haven't enabled it yet, you can do it using the following command:
+
+```
+Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All
+```
+
+After running this command a restart of the Windows machine is required.
+
+:information_source: The VM provider used by podman (Hyper-V or WSL) can be configured in the file 
+`%APPDATA%/containers/containers.conf`. More on that later.
+
+### Get the source code
+
+Open a Windows Terminal and run the following command:
+
+```
+git config --global core.autocrlf false
+```
+
+This configures git so that it does **not** automatically convert LF to CRLF. Files are expected to use the Unix LF rather 
+Windows CRLF in the podman git repository.
+
+Then run the command to clone podman git repository:
+
+```
+git clone https://github.com/containers/podman
+```
+
+This will create the folder `podman` in your current directory and clone podman git repository in it.
+
+### Build the podman client for windows
+
+The podman client for Windows can be build with the PowerShell script [winmake.ps1](https://github.com/containers/podman/blob/main/winmake.ps1).
+
+On Windows computers, the ExecutionPolicy is set `Restricted` by default: running scripts is not allowed. You can verify 
+the ExecutionPolicy on your machine with this commmand:
+
+```
+Get-ExecutionPolicy
+```
+
+If the command returns `Restricted` you should change the ExecutionPolicy to `RemoteSigned`:
+
+```
+Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
```

Comment:
you prob need to note what the ramifications of this are.

Reply:
I think we you change something like this, it is prob a good idea to outright let people they are theoretically changing the security level of their machine; given that the default was to be tighter.

Reply:
@TomSweeneyRedHat wdyt?

Reply:
A note saying something to that effect would be good.  When ever a a change is made system wide, especially if it changes security settings, it should be highlighted.